modules/opengl/glsl/utils/structs.glsl

```diff
@@ -120,6 +120,19 @@ struct LightParameters {
     float specularExponent;
 };
 
+struct Material {
+    vec3 ambient;
+    vec3 diffuse;
+    vec3 specular;
```

Comment:
that about the exponent?

Reply:
Can add that one. Didn't want to have a duplicate of the `LightParameters` at the moment.

Reply:
Could also do that later I guess...

Reply:
No point adding it if we aren't setting/using it

Reply:
Perhaps this is MaterialColors and then there are other structs for each material model. Exponent is not used in all material models